webapp/tests/karma/ts/services/xml-forms.service.spec.ts

```diff
@@ -1112,6 +1186,9 @@ describe('XmlForms service', () => {
         })
         .catch(err => {
           expect(err.message).to.equal(`The form "${internalId}" doesn't have an xform attachment`);
+          expect(error.callCount).to.equal(1);
+          expect(error.args[0][0]).to.equal('Error in XMLFormService getDocAndFormAttachment : ');
+          expect(error.args[0][1]).to.equal(`The form "${internalId}" doesn't have an xform attachment`);
         });
```

Comment:
Please add a test where `dbGetAttachment` rejects with an error that is not a 404 and assert that a feedback doc is added.

Reply:
ok